.github/workflows/run_tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
interesting to know needing `'`. maybe better to have on all of them to be consistent.

Reply:
confirmed the fix.